client/my-sites/current-site/notice.jsx

```diff
@@ -105,7 +105,7 @@ export class SiteNotice extends React.Component {
 		return (
 			<SidebarBanner
 				ctaName="free-to-paid-sidebar"
-				ctaText={ translate( 'Upgrade' ) }
+				ctaText={ translate( 'Upgrade', { context: 'Upgrade from one plan to another' } ) }
```

Comment:
:disappointed: 4 existing translations will be lost with this change.
If the string must change, the following option has already been translated **37** times:
`translate( 'Upgrade', { context: 'Domain mapping suggestion button with plan upgrade'} )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/context-to-upgrade-label)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).